go-runtime/compile/external-module-template/_ftl/go/modules/{{ range .NonMainModules }}{{ push .Name . }}{{ end }}/external_module.go

```diff
@@ -7,6 +7,11 @@ import (
 {{- range $import, $alias := (.|imports)}}
   {{if $alias}}{{$alias}} {{end}}"{{$import}}"
 {{- end}}
+{{- $sumTypes := sumTypes $}}
```

Comment:
I find it a bit clearer to use `$ | sumTypes`, but I think it's personal preference, so up to you.

Reply:
Ahh sounds good, I updated it to match yours for consistency with the rest of the file